src/eth-query.ts

```diff
@@ -0,0 +1,656 @@
+import createRandomIdFactory from 'json-rpc-random-id';
```

Comment:
Since we are backfilling the types for this library here, I tested building the project to make sure that this doesn't show up to prevent the same sort of error that we are having with `eth-query`, and while it does appear in the compiled JS, it does not appear in the `.d.ts`. So I think we're okay on this.